lib/taskana-core/src/test/java/acceptance/task/transfer/TransferTaskAccTest.java

```diff
@@ -435,4 +435,220 @@ void should_NotSetTheTransferFlagWithinBulkTransfer_When_SetTransferFlagNotReque
+    assertThat(transferredTask.getDomain()).isEqualTo(wb.getDomain());
+    assertThat(transferredTask.getModified().isBefore(before)).isFalse();
+    assertThat(transferredTask.getOwner()).isEqualTo("teamlead-1");
+  }
+
+  @WithAccessId(user = "admin")
+  @Test
+  void should_SetTransferFlagToFalse_When_WithOwnerAndWorkbasketIdGiven() throws Exception {
+    taskService.transferWithOwner(
+        "TKI:000000000000000000000000000000000010",
+        "WBI:100000000000000000000000000000000006",
+        "user-1-1",
+        false);
+
+    Task transferredTask = taskService.getTask("TKI:000000000000000000000000000000000010");
+    assertThat(transferredTask).isNotNull();
+    assertThat(transferredTask.isTransferred()).isFalse();
+    assertThat(transferredTask.isRead()).isFalse();
+    assertThat(transferredTask.getState()).isEqualTo(TaskState.READY);
+    assertThat(transferredTask.getOwner()).isEqualTo("user-1-1");
+    assertThat(transferredTask.getWorkbasketSummary().getId())
+        .isEqualTo("WBI:100000000000000000000000000000000006");
+  }
+
+  @WithAccessId(user = "admin")
+  @Test
+  void should_NotSetTheTransferFlagWithinBulkTransferWithOwner_When_WorkbasketIdGiven()
+      throws Exception {
+    taskService.transferTasksWithOwner(
+        "WBI:100000000000000000000000000000000006",
+        List.of(
+            "TKI:000000000000000000000000000000000010",
+            "TKI:000000000000000000000000000000000011",
+            "TKI:000000000000000000000000000000000012"),
+        "user-1-1",
+        false);
+
+    List<TaskSummary> transferredTasks =
+        taskService
+            .createTaskQuery()
+            .idIn(
+                "TKI:000000000000000000000000000000000010",
+                "TKI:000000000000000000000000000000000011",
+                "TKI:000000000000000000000000000000000012")
+            .list();
+
+    assertThat(transferredTasks).extracting(TaskSummary::isTransferred).containsOnly(false);
+  }
+
+  @WithAccessId(user = "admin")
+  @Test
+  void should_SetTransferFlagToFalse_When_WithOwnerAndWorkbasketKeyAndDomainGiven()
+      throws Exception {
+    taskService.transferWithOwner(
+        "TKI:000000000000000000000000000000000011", "USER-1-1", "DOMAIN_A", "user-1-1", false);
+
+    Task transferredTask = taskService.getTask("TKI:000000000000000000000000000000000011");
+    assertThat(transferredTask).isNotNull();
+    assertThat(transferredTask.isTransferred()).isFalse();
+    assertThat(transferredTask.isRead()).isFalse();
+    assertThat(transferredTask.getState()).isEqualTo(TaskState.READY);
+    assertThat(transferredTask.getOwner()).isEqualTo("user-1-1");
+    assertThat(transferredTask.getWorkbasketSummary().getId())
+        .isEqualTo("WBI:100000000000000000000000000000000006");
+  }
+
+  @WithAccessId(user = "admin")
+  @Test
+  void should_NotSetTheTransferFlagWithinBulkTransferWithOwner_When_WorkbasketKeyAndDomainGiven()
```

Comment:
The test name is confusing. The transfer flag is set to false, as specified in the test

Reply:
I have changed it to "should_SetTransferFlagAsSpecifiedWithinBulkTransferWithOwner_When_WorkbasketKeyDomainGiven"